docs/extensibility/60-form-widgets.md

```diff
@@ -72,6 +73,32 @@ See the following examples:
 <img src="./assets/form-widgets/Dropdown.png" alt="Example of a dropdown text widget with a tooltip" style="border: 1px solid #D2D5D9">
 <img src="./assets/form-widgets/Dropdown2.png" alt="Example of a dropdown text widget" style="border: 1px solid #D2D5D9">
 
+### `Switch`
+
+The `Switch` widgets render a switch button that is used to control boolean values.
+
+| Parameter         | Required | Type    | Description                                                                                                                                                             |
+| ----------------- | -------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **description**   | No       | string  | A string displayed in a tooltip when you hover over a question mark icon, next to the input's label. The default value is taken from the CustomResourceDefintion (CRD). |
+| **disableOnEdit** | No       | boolean | Disables a field in edit mode, defaults to `false`.                                                                                                                     |
```

Comment:
```suggestion
| **disableOnEdit** | No       | boolean | Disables a switch in edit mode, defaults to `false`.                                                                                                                     |
```

Reply:
addressed in [a7c3a09](https://github.com/kyma-project/busola/pull/3211/commits/a7c3a09f0e82f38c9512ffe5e4a79538ba5f7428)